packages/types/src/types/provider/openshift/Namespace.ts

```diff
@@ -1,11 +1,9 @@
 import { V1Namespace } from '../../k8s/Namespace';
 
-import { Resource } from './Resource';
+import { OpenshiftResource } from './Resource';
 
 // https://github.com/kubev2v/forklift/tree/main/pkg/controller/provider/web/ocp/namespace.go
-export interface Namespace extends Resource {
+export interface OpenShiftNamespace extends OpenshiftResource {
```

Comment:
The original assumption was that:
1. file name follows the main type.
2. we use hierarchical structure i.e. `openshift/Namespace`  instead of  prefix based i.e. `OpenShiftNamespace`